test/integration/PublishFlows.test.ts

```diff
@@ -198,6 +199,111 @@ describe('Publish tests', async () => {
 
   delay(19000)
 
+  it('should publish a dataset with fixed price (create NFT + Datoken + fixed price) using createAsset() fn', async () => {
+    const fixedPriceDdo: DDO = { ...genericAsset }
+    const ownerAddress = publisherAccount
+    const asset = await createAsset(
+      'test asset',
+      'TEST',
+      ownerAddress,
+      assetUrl,
+      1, // template 1 on dev network
+      fixedPriceDdo,
+      true, // encrypted ddo
+      providerUrl,
+      ZERO_ADDRESS, // provider fee token
+      aquarius
+    )
+
+    console.log('Published asset, ddo id:', asset)
+
+    // const nftParams: NftCreateData = {
```

Comment:
shall we use this code after merging this branch with main?

Reply:
not the console.log :-) I've cleaned the test. If you mean the new function createAsset, yes, its an option, or we can do it all in many steps as we we're doing before (like some of the other tests)